routes/api/v1/favorites.js

```diff
@@ -52,14 +52,25 @@ router.post('/', async (request, response) => {
 });
 
 router.get('/', async (request, response) => {
-  database('favorites')
+  database('favorites').select()
       .then((favorites) => {
         if (favorites.length) {
           response.status(200).json(favorites);
         } else {
-          response.status(200).json({ message: 'No favorites found!' });
+          response.status(200).json(favorites);
+        }
```

Comment:
Since the same code is executed whether or not there are favorites, we can get rid of the if/else statements